src/Analyzer/FunctionNode.h

```diff
@@ -1,12 +1,14 @@
 #pragma once
 
 #include <memory>
-#include <Core/IResolvedFunction.h>
+#include <AggregateFunctions/IAggregateFunction.h>
+#include <Analyzer/ConstantValue.h>
 #include <Analyzer/IQueryTreeNode.h>
 #include <Analyzer/ListNode.h>
-#include <Analyzer/ConstantValue.h>
-#include <Common/typeid_cast.h>
 #include <Core/ColumnsWithTypeAndName.h>
+#include <Core/IResolvedFunction.h>
+#include <Common/typeid_cast.h>
```

Comment:
In other files we first include Common, then Core, then other subsystems, then Analyzer.